pinc/dpsession.inc

```diff
@@ -71,6 +73,17 @@ function dpsession_resume()
     {
         _update_user_activity_time( FALSE );
     }
+
+    // stash the http_referer from the user's very first page load of the site
```

Comment:
Elsewhere in this PR we use "referrer" rather than "referer". Is there some reason for this inconsistency?

Reply:
Funny you should ask: https://en.wikipedia.org/wiki/HTTP_referer

Reply:
The `http_referer` cookie is being set here in `pinc/dpsession.inc`. The contents of that cookie are being saved in the `non_activated_users.http_referrer` column in `accounts/addproofer.php` and, upon activation, moved into the `users` table in `accounts/activate.php`.

Reply:
Got it. Thanks. Consistency of spelling would be nice... Maybe our internal field should be http_referer, too?

Reply:
I thought about that, but then decided it would be very weird to see right next to the other `referrer` field. I opted to push the weirdness at the point of consumption (when we get the HTTP_REFERER from Apache) rather than propagate that throughout the code and database. I'm not sure there is a good decision - I can easily see both sides of it.